pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -143,6 +143,7 @@ func (pl *DefaultPreemption) SelectVictimsOnNode(
 	nodeInfo *framework.NodeInfo,
 	pdbs []*policy.PodDisruptionBudget) ([]*v1.Pod, int, *framework.Status) {
 	var potentialVictims []*framework.PodInfo
+	logger := klog.FromContext(ctx)
```

Comment:
Can this be migrate to https://github.com/kubernetes/kubernetes/pull/116835, I think they're related. Preemption plugin will call the defaultPreemption.

Reply:
ok, I'll do that in a few days.